<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleEBCUPage2</class>
 <widget class="QWidget" name="VehicleEBCUPage2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	font: 20px, &quot;微软雅黑&quot;;
	color:white;
background-color:blue;

}
QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 128);
	border-top: 2px solid white;	
	border-left: 2px solid white;
border-radius:8px;
}</string>
  </property>
  <widget class="QLabel" name="ASPM1_2lbl">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>BC压(bar)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPageBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30px;
color:black;
background-color:orange;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>︿</string>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM1_1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPM1lbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPM1_1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP1_1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>135</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>停放制动
缸压(bar)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadTC1lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACPTC2_1lbl">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP2_2lbl">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>275</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC1_2lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadMP2lbl">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPTC1lbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadTC2lbl">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC2_2lbl">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM2_1lbl">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadMP2lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPMP1_2lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPTC1_1lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>空簧压力
(bar)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM1_2lbl">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_13">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;
border-radius:10px;
font:14px;
color:yellow;
background-color:darkgray;</string>
   </property>
   <property name="text">
    <string>






第2页/共2页</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPTC2lbl">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>载荷(T)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPTC1_2lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>345</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>压缩机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_12">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>415</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP2_1lbl">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPTC2_2lbl">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACPTC1_1lbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>乘车率(%)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPM2_2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPMP2_1lbl">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>205</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadTC2lbl">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC2_1lbl">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM2_2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LoadMP1lbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPMP1_1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPTC2_1lbl">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP1_2lbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPMP2lbl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPM2_1lbl">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC1_1lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PLoadTC1lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASPMP2_2lbl">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>485</y>
     <width>740</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPTC1bg1lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPTC2bg1lbl">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>MR压(bar)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPMP1bg1lbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPM1bg1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPM2bg1lbl">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MRPMP2bg1lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl1_8</zorder>
  <zorder>lbl1_10</zorder>
  <zorder>lbl1_14</zorder>
  <zorder>lbl1_12</zorder>
  <zorder>lbl1_11</zorder>
  <zorder>lbl1_9</zorder>
  <zorder>lbl1_7</zorder>
  <zorder>lbl1_13</zorder>
  <zorder>ASPM1_2lbl</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>PLoadMP1lbl</zorder>
  <zorder>NextPageBtn</zorder>
  <zorder>BCPM1_1lbl</zorder>
  <zorder>MRPM1lbl</zorder>
  <zorder>PLoadM1lbl</zorder>
  <zorder>ASPM1_1lbl</zorder>
  <zorder>BCPMP1_1lbl</zorder>
  <zorder>lbl1_6</zorder>
  <zorder>LoadTC1lbl</zorder>
  <zorder>ACPTC2_1lbl</zorder>
  <zorder>BCPMP2_2lbl</zorder>
  <zorder>BCPTC1_2lbl</zorder>
  <zorder>PLoadMP2lbl</zorder>
  <zorder>MRPTC1lbl</zorder>
  <zorder>PLoadTC2lbl</zorder>
  <zorder>BCPTC2_2lbl</zorder>
  <zorder>LoadM2lbl</zorder>
  <zorder>BCPM2_1lbl</zorder>
  <zorder>LoadMP2lbl</zorder>
  <zorder>ASPMP1_2lbl</zorder>
  <zorder>ASPTC1_1lbl</zorder>
  <zorder>lbl1</zorder>
  <zorder>BCPM1_2lbl</zorder>
  <zorder>MRPTC2lbl</zorder>
  <zorder>lbl1_3</zorder>
  <zorder>ASPTC1_2lbl</zorder>
  <zorder>lbl1_5</zorder>
  <zorder>LoadM1lbl</zorder>
  <zorder>BCPMP2_1lbl</zorder>
  <zorder>PLoadM2lbl</zorder>
  <zorder>MRPM2lbl</zorder>
  <zorder>ASPTC2_2lbl</zorder>
  <zorder>ACPTC1_1lbl</zorder>
  <zorder>lbl1_4</zorder>
  <zorder>ASPM2_2lbl</zorder>
  <zorder>ASPMP2_1lbl</zorder>
  <zorder>LoadTC2lbl</zorder>
  <zorder>BCPTC2_1lbl</zorder>
  <zorder>BCPM2_2lbl</zorder>
  <zorder>LoadMP1lbl</zorder>
  <zorder>ASPMP1_1lbl</zorder>
  <zorder>ASPTC2_1lbl</zorder>
  <zorder>BCPMP1_2lbl</zorder>
  <zorder>MRPMP2lbl</zorder>
  <zorder>ASPM2_1lbl</zorder>
  <zorder>BCPTC1_1lbl</zorder>
  <zorder>PLoadTC1lbl</zorder>
  <zorder>ASPMP2_2lbl</zorder>
  <zorder>MRPMP1lbl</zorder>
  <zorder>MRPTC1bg1lbl</zorder>
  <zorder>MRPTC2bg1lbl</zorder>
  <zorder>lbl1_15</zorder>
  <zorder>MRPMP1bg1lbl</zorder>
  <zorder>MRPM1bg1lbl</zorder>
  <zorder>MRPM2bg1lbl</zorder>
  <zorder>MRPMP2bg1lbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
